django/core/urls.py

```diff
@@ -42,23 +106,12 @@
     # https://docs.wagtail.io/en/v2.9.2/reference/contrib/sitemaps.html
     path("sitemap.xml", sitemap),
     path("robots.txt", include("robots.urls")),
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+]
 
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 urlpatterns += feeds.urlpatterns()
-
-if settings.DEPLOY_ENVIRONMENT.is_development:
-    # serve static files from development server
-    # https://docs.djangoproject.com/en/3.0/howto/static-files/#serving-static-files-during-development
-    urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-
-if not settings.DEPLOY_ENVIRONMENT.is_production:
-    import debug_toolbar
-
-    urlpatterns = [
-        path("argh/", handler500, name="error"),
-        path("make-error/", views.make_error),
-        path("__debug__/", include(debug_toolbar.urls)),
-    ] + urlpatterns
+urlpatterns += get_dev_urls()
```

Comment:
I'm pretty sure this will throw an error in prod e.g., `[1, 2, 3] + None`. Would need to fix `get_dev_urls` and `get_debug_urls` to `return []` in the else clause

It might be cleaner / easier to see what's happening by keeping the conditional guard where we mutate urlpatterns but don't feel too strongly about it either way

Reply:
Thanks for catching that. Conditionals at the bottom is a bit more concise and maybe easier to follow, I'll go with that